matter/efr32/efr32mg24/BRD4186A/autogen/sl_device_init_clocks.c

```diff
@@ -33,12 +33,12 @@
 
 sl_status_t sl_device_init_clocks(void)
 {
-    CMU_ClockSelectSet(cmuClock_SYSCLK, cmuSelect_HFXO);
+    CMU_ClockSelectSet(cmuClock_SYSCLK, cmuSelect_HFRCODPLL);
```

Comment:
Did the 4186A really swapped from HFXO to HFRCODPLL ?

Reply:
adding the DPLL componenent changes the clock selection